src/utils/input.ts

```diff
@@ -26,3 +29,31 @@ export const generateRadioItems = (items: string[]): RadioItem[] =>
     id: item,
     value: item,
   }));
+
+/**
+ * Get the translated name of an item along with the language of the item.
+ *
+ * @param {TafsirInfo | AvailableTranslation} item the item that we want to generate the label for.
+ * @returns {String}
+ */
+export const getTranslatedLabelWithLanguage = (item: TafsirInfo | AvailableTranslation): string =>
+  `${capitalize(item.languageName)} - ${item.translatedName.name}`;
+
+/**
+ * Convert a TafsirInfo or AvailableTranslation items into combobox items.
+ *
+ * @param {TafsirInfo[] | AvailableTranslation[]} items
+ * @returns  {DropdownItem[]}
+ */
+export const itemsToComboboxItems = (
```

Comment:
Have a mixed opinion here for 2 reasons: 

1) the name is misleading, initially I wanted to suggest naming the function "arrayToComboboxItems" but this function is not a generic one, it's only for tafsirs and available translations. Which leads me to 2)

2) Since this function is only for tafsirs and available translations schema, imo it shouldn't be a util. It's ok to have the code duplicated in both files.

Reply:
Okay no strong opinion here, just wanted to save duplicating the data but can revert no probs.